### ISSUE TITLE
Importing map problem. Getting error missing texture

### ISSUE THREAD

**Issue by sharondenys** [Seanba/Tiled2Unity#186]
Hi I have this issue when I try to import my files to unity.

-sample_map.tmx
-Rougelike.tsx

and my sprite sheet

-roguelikeSheet_transparent.png

These are my screenshots.

<img width="615" alt="screen shot 2019-02-21 at 2 46 01 pm" src="https://user-images.githubusercontent.com/33921912/53149144-d8a63700-35e7-11e9-91a0-e0e0cd7df8ee.png">

<img width="596" alt="screen shot 2019-02-21 at 2 45 57 pm" src="https://user-images.githubusercontent.com/33921912/53149166-eb207080-35e7-11e9-8608-ec886ab612d7.png">

I downloaded the asset from Kenny's asset. It comes with a complete sample map and a transparent sprite sheet. What does the error mean by missing textures? Because I've imported all the file needed.

It would be great if you could help.

**Comment by Seanba** [comment]
Hi there, @sharondenys. Does your `sample_map.tmx` work when you try to open it in Tiled? If SuperTiled2Unity is having trouble finding the referenced files then I would imagine Tiled can't find them either.

If I had to guess, your problem is that `roguelikeSheet_transparent.png` is not in the location that `Rougelike.tsx` is expecting it. The relation of your files in Unity should be something like this ...

`SomeFolder/Rougelike.tsx`
`SomeFolder/roguelike-pack/Spritesheet/roguelikeSheet_transparent.png`

In other words, you have to keep the relative paths to your TMX, TSX, and PNG files intact as you import them in Unity.

**Comment by Seanba** [comment]
Also, this is the Tiled2Unity depot, not the SuperTiled2Unity one :)

**Comment by sharondenys** [comment]
Hello, 

I managed to fix the issue after playing around with it. It is the file location that went wrong. Thanks!

**Comment by Seanba** [comment]
You're welcome. Good luck with your game.
